civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5024,8 +5024,15 @@ function _civihr_employee_portal_should_do_onboarding($account) {
 
   $canViewOnboardingForm = node_access('view', $onboardingWebform, $account);
   $submissions = WebformHelper::getUserSubmissionsByTitle($account, $formName);
+  $user_wants_to_test = isset($_GET['testing']);
+  // user who can edit it's contact doesn't need this form
+  if (!$user_wants_to_test && user_access('edit my contact')) {
+    return false;
+  }
```

Comment:
I can't see this as a requirement either in the ticket or in the specification. Where did it come from?

If it is really necessary, what do you think of moving it right to the beginning of this function? This way, we could avoid all the unnecessary calls to load the webform since the user won't have access to it anyway

Reply:
Is related to this: "HR Admin, Root and Local HR Admin users should not have to fill the onboarding wizard at any point. They can update their data from the HR Admin side." The ones who can "edit my contact" are not required to do the "Onboarding Wizard".

"If it is really necessary, what do you think of moving it right to the beginning of this function?" Totally agree, thanks.